connectors-e2e-test/connectors-e2e-test-base/src/main/java/io/camunda/connector/e2e/BpmnFile.java

```diff
@@ -26,14 +26,18 @@
 
 public class BpmnFile {
 
-  private BpmnModelInstance bpmnModelInstance;
-
+  private final BpmnModelInstance bpmnModelInstance;
   private File bpmnFile;
 
   public BpmnFile(BpmnModelInstance bpmnModelInstance) {
     this.bpmnModelInstance = bpmnModelInstance;
   }
 
+  public BpmnFile(File bpmnFile) {
```

Comment:
We load the process from file in the Agentic AI e2e tests and don't need any replacements. Without this constructor we'd need to:

1. load it into a model instance
2. create a `BpmnFile` using the model instance
3. call `BpmnFile.writeToFile` to create a temporary file (containing the same contents as the input one)
4. call `BpmnFile.apply`

With this constructor we can simply do:

```java
File process = ...;
var updatedModel =
    new BpmnFile(process)
        .apply(elementTemplate, "elementId", new File(tempDir, "updated.bpmn"));
```